### PR TITLE
feat: full functionality added

### PR DESCRIPTION
**Pull request by GuramSalia**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -38,10 +38,12 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.springframework.cloud</groupId>
-            <artifactId>spring-cloud-starter-config</artifactId>
-        </dependency>
+
```
Comment (SiarheiBor) [inline pom.xml:41]: remove instead of comment

File: pom.xml [hunk 2]
```diff
@@ -121,22 +123,29 @@
...
-        <dependency>
-            <groupId>org.springframework.cloud</groupId>
-            <artifactId>spring-cloud-starter-netflix-eureka-client</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>org.springframework.cloud</groupId>
-            <artifactId>spring-cloud-starter-openfeign</artifactId>
-            <version>4.1.1</version>
-        </dependency>
+        <!-- related to microservices communicating via rest -->
```
Comment (SiarheiBor) [inline pom.xml:126]: remove instead of comment

File: pom.xml (hunk 1 above)
Comment (GuramSalia) [reply pom.xml:41]: deleted

File: pom.xml (hunk 2 above)
Comment (GuramSalia) [reply pom.xml:126]: removed

**Pull request merged**